tests/multi_bluetooth/ble_gap_advertise.py

```diff
@@ -42,13 +42,13 @@ def irq(ev, data):
                     adv_data = bytes(data[4])
                 else:
                     if adv_data != data[4]:
-                        adv_data = "MISMATCH"
+                        adv_data = b"MISMATCH"
         elif ev == _IRQ_SCAN_COMPLETE:
             finished = True
 
     ble.config(rxbuf=2000)
     ble.irq(irq)
-    ble.gap_scan(2 * ADV_TIME_S * 1000, 10000, 10000)
+    ble.gap_scan(2 * ADV_TIME_S * 1000, 30000, 30000)
```

Comment:
Is this change still needed now that the scan-response data issue is fixed?

Reply:
Oh nope, I had it in my head that 30000 is the minimum (it isn't, rather it's NimBLE's default for "fast" scanning). Removed this change.